app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def profile
+    @user = User.find(params[:id])
```

Comment:
- [ ] The User#index view requires a list of events created by the user so you need to define a `@created_events` variable with the list of events in this action so it can be used in the view.